internal/testdata/snapshots/input/testspecial/foo_test.go

```diff
@@ -0,0 +1,3 @@
+package testspecial
+
+func TestFoo_Whitebox() { Foo() }
```

Comment:
As of the first commit, the snapshot test is (incorrectly) not generating `foo_test.go` in the output directory. This is fixed in the second commit.